planner/core/explain.go

```diff
@@ -386,6 +386,18 @@ func (p *PhysicalIndexJoin) ExplainNormalizedInfo() string {
 	return p.explainInfo(true)
 }
 
+// ExplainInfo implements Plan interface.
+func (p *PhysicalApply) ExplainInfo() string {
+	buffer := new(bytes.Buffer)
+	if p.CanUseCache {
+		buffer.WriteString("cache: ON, ")
+	}
+	if p.Concurrency > 1 {
+		buffer.WriteString(fmt.Sprintf("concurrency: %v, ", p.Concurrency))
+	}
```

Comment:
How about putting concurrency info in `execution info` for unifying?

Reply:
Updated. PTAL.